ivy_tests/test_ivy/test_frontends/test_torch/test_spectral_ops.py

```diff
@@ -0,0 +1,35 @@
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.test_frontends.test_torch.test_tensor import _requires_grad
+from hypothesis import strategies as st
+from ivy_tests.test_ivy.helpers.testing_helpers import handle_frontend_test
+
+
+@handle_frontend_test(
+    fn_tree="torch.hann_window",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric")
+    ),
+    periodic=st.booleans(),
+    dtype=helpers.get_dtypes("numeric", full=False, none=True),
+    requires_grad=_requires_grad(),
+)
+def test_torch_hann_window(
+    periodic,
+    dtype_and_x,
+    dtype,
+    num_positional_args: 2,
```

Comment:
`num_positional_args` does not seem to be used

Reply:
removed it😁